<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cpp_qt_chartsClass</class>
 <widget class="QMainWindow" name="cpp_qt_chartsClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>806</width>
    <height>580</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>cpp_qt_charts</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QChartView" name="chartDisplay"/>
    </item>
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>0</height>
       </size>
      </property>
      <property name="title">
       <string>Chart controls</string>
      </property>
      <widget class="QDoubleSpinBox" name="spinX">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>151</width>
         <height>22</height>
        </rect>
       </property>
       <property name="minimum">
        <double>-10000.000000000000000</double>
       </property>
       <property name="maximum">
        <double>10000.000000000000000</double>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="spinY">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>70</y>
         <width>151</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>Add Point</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnAddValue">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>100</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Add Point</string>
       </property>
      </widget>
      <widget class="QPlainTextEdit" name="memoPoints">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>140</y>
         <width>151</width>
         <height>261</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QGraphicsView</extends>
   <header>QtCharts</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="cpp_qt_charts.qrc"/>
 </resources>
 <connections/>
</ui>
